presto-parser/src/main/antlr4/com/facebook/presto/sql/parser/SqlBase.g4

```diff
@@ -59,6 +59,8 @@ statement
         (GRANT OPTION FOR)?
         (privilege (',' privilege)* | ALL PRIVILEGES)
         ON TABLE? qualifiedName FROM grantee=identifier                #revoke
+    | SHOW GRANTS
```

Comment:
I followed the same syntax that `SHOW GRANT` command in Hive has. But I guess, `ON ALL` seems redundant; I can remove it.